net.sourceforge.roccat.roccat-tools.yml

```diff
@@ -0,0 +1,68 @@
+app-id: net.sourceforge.roccat.roccat-tools
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=all
+rename-icon: roccat
+
```

Comment:
make sure to add a `cleanup` to remove things like development files (headers, static libraries) that were installed.